backend/dal/Repositories/NoteRepository.cs

```diff
@@ -95,17 +95,15 @@ public IEnumerable<PimsNote> GetActivityNotes(long entityId)
                 .Where(x => x.ActivityInstanceId == entityId && (x.IsDisabled ?? false) == false).Select(x => x.Note).ToList();
         }
 
-        public void DeleteActivityNotes(long noteId)
+        public void DeleteActivityNotes(long entityId)
         {
-            var activityNotes = this.Context.PimsActivityInstanceNotes.Where(x => x.NoteId == noteId).ToList();
+            var activityNotes = this.Context.PimsActivityInstanceNotes.Where(x => x.ActivityInstanceId == entityId).ToList();
             if (activityNotes.Any())
             {
-                foreach (var note in activityNotes)
+                foreach (var activityNote in activityNotes)
```

Comment:
@asanchezr please confirm this.
I didn't see requirements for this and if a note still exists in an activity the activity can't be deleted.
If we need notes for auditing is the history table not sufficient?

Reply:
I didn't code the delete notes feature. That was @Maheshbabu28 story - he can expand on this decision

Reply:
But the presence of **isDisabled** flag seems to indicate that they would want a "soft" delete - or else why would we need it?

Reply:
I think doug adds it as a default to basically all of our entities.

Reply:
confirmed, we do not need to soft delete notes. I will remove the logic that sets that flag during deleting in a future story.